packages/compass-app-stores/package.json

```diff
@@ -34,7 +34,7 @@
   },
   "types": "./dist/index.d.ts",
   "scripts": {
-    "bootstrap": "npm run postcompile",
```

Comment:
For a moment I couldn't figure out how this worked before then I realised postcompile was already removed in a previous commit 👍

Reply:
Yeah, sorry, this one should've been part of automated changes, but I messed up the script a bit, didn't want to re-run the whole thing, and so had to split it to a separate commit 🙈